extra/pdf/layout/layout.factor

```diff
@@ -359,7 +359,7 @@ TUPLE: pdf-info title timestamp producer author creator ;
     pdf-info new
         now >>timestamp
         "Factor" >>producer
-        "USER" os-env "unknown" or >>author
+        "USER" os-env \ developer-name get-global "unknown" or or >>author
```

Comment:
I wonder if we should instead have symbols that are for the pdf-author, instead of trying to guess this way.

```factor
SYMBOLS: pdf-producer pdf-author pdf-creator ;
```